doc/whats_new.rst

```diff
@@ -131,6 +131,9 @@ Enhancements
    - Add option to show ``indicator features`` in the output of Imputer.
      By `Mani Teja`_.
 
+   - Reduce the memory usage of :func:`utils.mean_variance_axis` and  :func:`utils.incr_mean_variance_axis` 
```

Comment:
I think I still need to make `assign_rows_csr` to support fused types, which can benefit #6430 a lot.
And after that, we may add something about estimators in `whats_new`?